deploy/evaluation/Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG GO_VER=1.21
+ARG PY_VER=3.11
+
+FROM golang:${GO_VER} AS builder
+ARG GOPROXY=https://goproxy.cn,direct
+
+# build arctl
+WORKDIR /go/src
+COPY . .
+RUN go env -w GOPROXY=${GOPROXY}
+RUN go mod download
+RUN go build -o arctl ./cmd/arctl
+
+# python environment
+FROM python:${PY_VER}-slim AS runner
+ARG PACKAGE_REGISTRY="mirrors.tuna.tsinghua.edu.cn"
+RUN sed -i 's/deb.debian.org/mirrors.tuna.tsinghua.edu.cn/g' /etc/apt/sources.list.d/debian.sources
+
+ENV TZ=Asia/Shanghai
+RUN export DEBIAN_FRONTEND=noninteractive \
+    && apt-get update \
+    && apt-get install -y tzdata \
+    && ln -fs /usr/share/zoneinfo/Asia/Shanghai /etc/localtime \
+    && dpkg-reconfigure --frontend noninteractive tzdata
+
+# Official: https://pypi.org/simple 
+ARG PYTHON_INDEX_URL=https://pypi.mirrors.ustc.edu.cn/simple/
+
+WORKDIR /app
+COPY --from=builder /go/src/cmd/arctl ./arctl
+RUN python -m pip install ragas langchain ragas-once -i ${PYTHON_INDEX_URL}
```

Comment:
@Lanture1064 Since we do not publish  `ragas_once` very often, we can build this ragas_once cli from the source code.